### PR TITLE
Make Play Panel dockable and floatable.

### PR DESCRIPTION
**Pull request by tomcucinotta**
The play panel contains a few small widgets, and it fits conveniently into the right or left docking area. I prefer to have it docked on the right, when I open it. I thought others might share a similar line of thinking, so here it is. I hope this can be merged. Years ago, when I was using MuseScore quite often again, I had also made a patch to dock the play pane. This is on current master though (6a85d228e).
Note: dockable means that, with this patch, you can still undock it and position it as an independent window, if you really like that way.

### CODE REVIEW

File: mscore/playpanel.ui [hunk 1]
```diff
@@ -1,557 +1,559 @@
...
+   <bool>true</bool>
   </property>
-  <property name="focusPolicy">
-   <enum>Qt::ClickFocus</enum>
+  <property name="features">
+   <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
-   <string>Play Panel</string>
+   <string>PlayPanel</string>
```
Comment (Jojo-Schmitz) [inline mscore/playpanel.ui]: Why this change? I believe this is a user visible string and translatable, and as such should not use CamelCase

File: mscore/playpanel.ui [hunk 2]
```diff
@@ -1,557 +1,559 @@
...
+            <widget class="QLabel" name="posLabel">
+             <property name="sizePolicy">
+              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+               <horstretch>0</horstretch>
+               <verstretch>0</verstretch>
+              </sizepolicy>
+             </property>
+             <property name="font">
+              <font>
+               <family>Arial black</family>
```
Comment (Jojo-Schmitz) [inline mscore/playpanel.ui]: Do we have to hardcode font names?

Comment (tomcucinotta) [reply mscore/playpanel.ui]: here I don't know why there was the font family, it was there already before my change. My change just aims at reducing slightly the font size of the big timer text, for a better use of the available space in the dock (esp. if you do dock the panel in half-top/bottom way).

I'm going to push a revised version.

File: mscore/playpanel.ui (hunk 1 above)
Comment (tomcucinotta) [reply mscore/playpanel.ui]: no reason, it just slipped out, let's keep the former translatable string of course!

**Comment by tomcucinotta** [comment]
See the tomcucinotta / dockable-play-panel-v2 branch with the revised patch.

**Comment by IsaacWeiss** [comment]
If you push the changes to this branch, then this pull request will update.

File: mscore/playpanel.ui (hunk 2 above)
Comment (Jojo-Schmitz) [reply mscore/playpanel.ui]: Ah, I see, changing a font's size also adds the font family

**Comment by tomcucinotta** [comment]
pushed with -f to this same branch as well.

**Comment by tomcucinotta** [comment]
just made a new push after rebase on current master, any further issue? thanks

**Comment by anatoly-os** [comment]
Looks good to merge. Let me check how it works.

**Comment by anatoly-os** [comment]
I've checked the behavior and it looks correct. @lasconic what do you think?
